### PR TITLE
ENYO-2709: Added sideways tooltip examples

### PR DESCRIPTION
**Pull request by Djspaceg**
Enyo-DCO-1.1-Signed-off-by: Blake Stephens blake.stephens@lge.com

### CODE REVIEW

File: src/moonstone-samples/src/TooltipSample.js [hunk 1]
```diff
@@ -23,10 +24,27 @@ module.exports = kind({
 				]}
 			]
 		},
+		{name: 'sidewaysTooltipButton', kind: TooltipDecorator, style: 'left: 40%; z-index: 1;', components: [
+			{kind: Button, content: 'Sideways Tooltip'},
+			{name: 'sidewaysTooltip', kind: Tooltip, position: 'left top', components: [
+				{name: 'sidewaysTooltipText', content: 'I\'m a tooltip on the side.', style: 'white-space: normal'}
```
Comment (viodragon2) [inline src/moonstone-samples/src/TooltipSample.js]: let's get rid of `content` since it's bound to input value. L22 as well for drag tooltip

**Comment by viodragon2** [comment]
How about let the Draggable Tooltip button be the central point of testing for position, so that we can also test `above` and `below` as well as all the other sideway tooltips. That way, it might not look so crowded as is. Just a suggestion.

**Comment by Djspaceg** [comment]
I'll spend a little more time on the sample and make it more coherent.

**Comment by Djspaceg** [comment]
Sample greatly cleaned up and is now simple looking again.

**Pull request merged**